themes/src/main/resources/theme/keycloak.v2/account/src/app/content/signingin-page/SigningInPage.tsx

```diff
@@ -60,12 +60,27 @@ type CredType = string;
 type CredTypeMap = Map<CredType, CredentialContainer>;
 type CredContainerMap = Map<CredCategory, CredTypeMap>;
 
+interface CredData {
```

Comment:
This interface `CredData` still contains some metadata specific to "recovery codes" credential. However it seems to me that this interface is not needed? Is it possible to remove this interface definition and also remove definition of `        const credData: CredData = JSON.parse(credential.credentialData!);` from line 271? It seems to me this won't break anything as `credData` defined on line 271 is not used anywhere?

Reply:
OK. Removing. Good catch.